docs/run/quantum-serverless.mdx

```diff
@@ -35,6 +35,8 @@ Once you have built a Qiskit Pattern, you can use Quantum Serverless to deploy i
 
 Here is an example of computing the expectation value using the Qiskit Runtime Estimator primitive. This Python script should be saved in your working directory. (Warning! All contents of the working directory will be shipped to the cluster for execution.)
 
+Please refer to our guides on how to configure your pattern to [accept input arguments](https://qiskit-extensions.github.io/quantum-serverless/getting_started/basic/02_arguments_and_results.html) and [handle external python dependencies](https://qiskit-extensions.github.io/quantum-serverless/getting_started/basic/03_dependencies.html).
```

Comment:
Should this be a different location? It breaks the flow of saying on line 36 "Here is an example".

Reply:
yeah sounds good!